lib/samples/display_dimensions/README.md

```diff
@@ -0,0 +1,40 @@
+# Display dimensions
+
+Display dimension features from a mobile map package.
+
+![Image showing the Display Dimensions sample](display_dimensions.png)
+
+## Use case
+
+Dimensions show specific lengths or distances on a map. A dimension may indicate the length of a land parcel, or the distance between two features, such as a fire hydrant and the corner of a building.
+
+## How to use the sample
+
+When the sample loads, it will automatically display the map containing dimension features from the mobile map package. The name of the dimension layer containing the dimension features is displayed in the controls box. Control the visibility of the dimension layer with the "Dimension Layer visibility" switch, and apply a definition expression to show dimensions greater than or equal to 450m in length using the "Definition Expression" switch.
+
+**Note**: the minimum scale range of the sample is set to 1:35000 to maintain readability of the dimension features.
+
+## How it works
+
+1. Create a `MobileMapPackage` specifying the path to the .mmpk file.
+2. Load the mobile map package (mmpk) with `mmpk.loadAsync()`.
+3. After the mmpk successfully loads, get the map from the mmpk and add it to the map view: `mapView.setMap(mmpk.getMaps().get(0))`.
+4. Loop through the map's layers to find the `DimensionLayer` and set the name of the layer to the UI with `dimensionLayer.getName()`.
```

Comment:
It doesn't look like the sample is doing the "name" thing.

Reply:
It does not do it in the Swift sample either. Should I remove this?

Reply:
Unless we can determine a reason why Swift skipped it, I think we should include it. Maybe the name is too long to fit? Or maybe @jenmerritt knows why the implementation diverged from the sample design.

Reply:
Yeah we shouldn't diverge from the design (even if another platform does) unless it's a desktop based design and we're diverging due to a UI reason or something else mobile specific. Or, if there is some other good reason that we can say is an exception. So I would display the name if this is possible.

Reply:
Layer name has been added above the switches

Reply:
Ok great - so didn't appear to be any reason why not? It's definitely cropping up in a few implementations these differences, but I do think we're doing the right thing sticking to the design.